app/webpacker/components/Tickets/TicketWorkbenches/CompetitionResultActionerView/MiscActions/EventsMergedData/EventsMergedDataContent.jsx

```diff
@@ -0,0 +1,152 @@
+              },
+            };
+          }
+          return roundData;
+        }),
+      );
+    },
+  });
+
+  if (isEventsMergedDataPending || isDeleteDataPending) return <Loading />;
+  if (isEventsMergedDataError) return <Errored error={eventsMergedDataError} />;
+  if (isDeleteDataError) return <Errored error={deleteDataError} />;
+
+  return (
+    <>
+      <Message warning>
+        Note: The table below shows if there are merged data for Result and Scramble. Importing
+        more data will override the existing inbox data, but not the merged results data. You
+        may remove the currently merged results data using the interface below.
+      </Message>
+      <Table celled striped compact="very" definition>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>Event Round</Table.HeaderCell>
+            <Table.HeaderCell colSpan="2">Has Merged Results</Table.HeaderCell>
+            <Table.HeaderCell colSpan="2">Has Merged Scrambles</Table.HeaderCell>
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {eventsMergedData.map((roundData) => (
+            <Table.Row key={roundData.round_id}>
+              <Table.Cell>{roundData.round_name}</Table.Cell>
+              <DataActioner
+                roundData={roundData}
+                model="Result"
+                deleteMutate={deleteResultsDataMutate}
+                competitionId={competitionId}
+              />
+              <DataActioner
+                roundData={roundData}
+                model="Scramble"
+                deleteMutate={deleteResultsDataMutate}
+                competitionId={competitionId}
+              />
+            </Table.Row>
+          ))}
+        </Table.Body>
+      </Table>
+      Please be careful removing data! Data in the above table is live.
+      Remove all results and scrambles only, does not affect persons:
+      {' '}
+      <Button color="red" as="a" onClick={() => setShowConfirm(true)}>X ALL</Button>
+      <Confirm
+        open={showConfirm}
+        content={`You are about to remove all entries from Results and Scrambles for ${competitionId}. THIS ACTION CANNOT BE UNDONE! Please confirm below if you're sure.`}
+        onCancel={() => setShowConfirm(false)}
+        onConfirm={() => {
+          deleteResultsDataMutate({ competitionId, model: 'All' });
+          setShowConfirm(false);
+        }}
+      />
+    </>
+  );
+}
+
+function DataActioner({
+  roundData, model, deleteMutate, competitionId,
+}) {
+  const [showConfirm, setShowConfirm] = useState();
+  const count = model === 'Result' ? roundData.count.result : roundData.count.scramble;
+
+  return (
+    <>
+      <Table.Cell>{count > 0 ? 'Yes' : 'No'}</Table.Cell>
+      <Table.Cell>
+        <Button
+          color="red"
+          onClick={() => setShowConfirm(true)}
+          compact
+          disabled={count === 0}
+        >
+          X
+        </Button>
+      </Table.Cell>
```

Comment:
Have you tried putting this button into the "Yes/No cell" above, and using `floated="right"`?

Reply:
Yes, but I felt this view was better. The other view had a bit more space in between and wasn't able to reduce it by adding `compact`.

Reply:
Actually it's not like way better, it's just slightly better. Having `floated="right"` is also fine if merging header columns is not a good practice.